lib/mayaUsd/resources/ae/usdschemabase/collectionMayaHost.py

```diff
@@ -307,19 +312,26 @@ class MayaStringListData(CollectionStringListData):
     def __init__(self, collection, isInclude: bool):
         super().__init__(collection, isInclude)
 
-    def addStrings(self, items: Sequence[AnyStr]):
+    def addStrings(self, items: Sequence[AnyStr]) -> bool:
         '''
         Add the given strings to the model.
         '''
         with _UsdUndoBlockContext(_AddItemsCommand.commandName):
-            super().addStrings(items)
+            return super().addStrings(items)
 
-    def removeStrings(self, items: Sequence[AnyStr]):
+    def removeStrings(self, items: Sequence[AnyStr]) -> bool:
         '''
         Remove the given strings from the model.
         '''
         with _UsdUndoBlockContext(_RemoveItemsCommand.commandName):
-            super().removeStrings(items)
+            return super().removeStrings(items)
+
+    def replaceStrings(self, oldString, newString) -> bool:
+        '''
+        Replace the old string with thew newString on the model.
+        '''
+        with _UsdUndoBlockContext(_ReplaceItemsCommand.commandName):
```

Comment:
This will end-up calling teh Maya version of the addStrings and removeStrings. Not sure how commands-within-commands will end-up showing up in the undo queue in Maya. We can try this out and test it and improve it later if need be.